css/amp.css

```diff
@@ -27,7 +27,7 @@ html {
   overflow-x: hidden !important;
 }
 html, body {
-  height: auto !important;
+  height: 100% !important;
```

Comment:
No idea, will have to test. But this is the only way to trigger iOS scrolling with the other css changes I made.